ansible-ipi-install/README.md

```diff
@@ -374,6 +377,17 @@ included when the OpenShift manifests are created.
 
 NOTE: By default this directory is empty. 
 
+## Pre-caching RHCOS Images
+By default, the playbook will pre-download RHCOS images prior to actual cluster
+deployment.  It places these images in an Apache web server on the provisioning
+host, and modifies `metal3-config.yaml` or `install-config.yaml` (depending on 
+whether the version is set to 4.3 or 4.4+) to instruct the bootstrap to
+download the images from that web server during deployment.  
+
+If you wish to force the bootstrap to download the RHCOS images from the Internet,
+set the `use_rhcos_image_cache` variable to false (in your hosts file, via a 
```

Comment:
Instead of forcing user to set value of `use_rhcos_image_cache` to false, lets set the variable inside node-prep/defaults/main.yml to `use_rhcos_image_cache: false` . This way in the inventory/hosts.sample file we can just have a commented line that shows #use_rhcos_image_cache: true with a brief description. 

Example:
#(Optional) Enable playbook to pre-download RHCOS images prior to cluster deployment. Default is false.
#use_rhcos_image_cache=true

Reply:
Can we add this also to the inventory/hosts.sample file?

Reply:
Ack.  I think it's there already.